WordPress/Classes/ViewRelated/Stats/Period Stats/Countries/Map/CountriesMapView.swift

```diff
@@ -41,4 +52,24 @@ class CountriesMapView: UIView, NibLoadable {
         label.font = WPStyleGuide.fontForTextStyle(.footnote)
         label.textColor = .neutral(shade: .shade70)
     }
+
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        colors = mapColors(traits: traitCollection)
+        if let countries = countries {
+            setData(countries)
+        }
+    }
+
+    private func mapColors(traits: UITraitCollection?) -> [UIColor] {
```

Comment:
The function argument `traits` is never used because the if statement uses the `traitCollection`. I also see the view `traitCollection` is passed as argument above. Can we remove the argument or use traits in the if statement? I don't know if the function needs to accept different traits than the view trait.